helix-view/src/editor.rs

```diff
@@ -913,6 +915,7 @@ impl Default for Config {
             popup_border: PopupBorderConfig::None,
             indent_heuristic: IndentationHeuristic::default(),
             jump_label_alphabet: ('a'..='z').collect(),
+            auto_closing_overtype: false,
```

Comment:
I'd prefer to rename it to `overtype` but I don't know how the maintainers feel about this

Reply:
Or it might be better to refactor `AutoPairConfig` into a struct (moving the current enum into a new one) and putting `overtype` under it.

Reply:
I made a new commit with the suggested changes. Instead of implimenting the `From` trait like original did, it felt more appropriate to add a `get_pairs` function that returns the enum value instead.